recipe/meta.yaml

```diff
@@ -29,16 +29,18 @@ requirements:
     - python
 
 test:
+  files:
+    - fib.pyx
```

Comment:
fib.pyx was used for run_test.py, which isn't being used anymore. could it be reinstated or (better) upstream tests run?

Reply:
I've rewritten run_test.py to use fib.pyx. I'll try to run the upstream tests too